cmd/submit.go

```diff
@@ -97,6 +99,28 @@ Your submission can be found online at %s
 	if submission.Iteration == 1 {
 		msg += `
 To get the next exercise, run "exercism fetch" again.
+
+`
+		rand.Seed( time.Now().UTC().UnixNano() )
+		phrases := []string{
+			"For bonus points",
+			"Don't stop now: The fun's just begun",
+			"Some tips to continue",
+		}
+		msg += fmt.Sprintf("## %s", phrases[rand.Intn(len(phrases))])
+		msg += `
+
+Did you get the tests passing and the code clean? If you want to, these are some
+additional things you could try:
+
+* Remove as much duplication as you possibly can.
+* Optimize for readability, even if it means introducing duplication.
+* If you've removed all the duplication, do you have a lot of conditionals? Try
+  replacing the conditionals with polymorphism, if it applies in this language.
+  How readable is it?
+
+Then please share your thoughts in a comment on the submission. Did this
+experiment make the code better? Worse? Did you learn anything from it?
 `
```

Comment:
Looking into text/template, it seems like it'll make the code less readable overall. It either:
* Adds overhead for creating the template plus a data structure (struct/map), while still requiring all the text in the source file, or
* Creates a run-time dependency on an external file using `ParseFiles`, which doesn't really make sense.

If anything it makes sense to define the static text as a const definite at the end of the file, eg:
```
    msg += fmt.Sprintf("## %s", phrases[rand.Intn(len(phrases))])
    msg += tips
...
const tips = `
(tips here)
`
```